config.json

```diff
@@ -734,6 +734,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 2
+      },
+      {
+        "slug": "zebra-puzzle",
+        "name": "Zebra Puzzle",
+        "uuid": "c7eec5da-4da5-48fb-aa11-7feaf13f2f3d",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
```

Comment:
I would consider this a hard problem, consider making the difficulty `8`

Reply:
I think that sounds reasonable. I'll make the change.